packages/framework/framework.js

```diff
@@ -1,8 +1,8 @@
 import Backdrop from '../backdrop/backdrop';
+import Breadcrumbs from '../breadcrumb/breadcrumb';
```

Comment:
Breadcrumb
I think, here too

Reply:
Fixed in 974ba475647f219bfce5520a56b77e3a5b92941d.